File: Team4.Seshat.Model/ViewModels/RegistrationRequest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Team4.Seshat.Model.ViewModels
{
    public class RegistrationRequest
    {
        public string Username { get; set; }
        public string Password { get; set; }
        public string ConfirmPassword { get; set; }
    }
}

File: front-end-ui/src/app/models/registration-request.ts
import { User } from "./user";

export class RegistrationRequest {
    constructor(username: string, password: string, confirmPassword: string) {
        this.username = username;
        this.password = password;
        this.confirmPassword = confirmPassword;
    }

    public username: string;
    public password: string;
    public confirmPassword: string;
}